Documentation/contributing/development/contributing_guide.rst

```diff
@@ -327,6 +327,11 @@ steps 1 to 2 times per day. Works best if done first thing in the working day.
 #. If this PR was opened by a non-committer (e.g. external contributor) please
    assign yourself to that PR and make sure to keep track the PR gets reviewed
    and merged. This may extend beyond your assigned week for Janitor duty.
+
+   If the contributor is a Cilium committer, then they are responsible for
```

Comment:
I assume that committer can add _only_ add `ready-to-merge` if all two conditions are satisfied:

- all tests passed, (or known flake is identified)
- receive all approvals from all assignees (automatically assigned from code owner based on round robin policy I assume).

While first point is under control, but second point is sometimes hard to keep track. Maybe that is why I rarely add `ready-to-merge` label :thinking:

Reply:
>I assume that committer can add only add ready-to-merge if all two conditions are satisfied

That is correct :+1: (as also state in point 4 [here](https://docs.cilium.io/en/v1.9/contributing/development/contributing_guide/#pull-requests-review-process-for-committers))